<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewAndUpdateProjectWindow</class>
 <widget class="QMainWindow" name="NewAndUpdateProjectWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>477</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>630</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Noto Sans CJK SC Regular</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>修改项目</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="projectNameLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>项目名称</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="projectNameLineEdit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>271</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>项目地址</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="projectAddressLineEdit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>100</y>
      <width>271</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>备注</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="remarkTextEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>441</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="savePushButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>390</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetPushButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>390</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>重置</string>
    </property>
   </widget>
   <widget class="QLabel" name="projectIdLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>121</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>【未知项目ID】</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
